MachineLearning_Engine/Compute/LoadCsv.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.MachineLearning;
+using BH.oM.MachineLearning;
+using BH.oM.Reflection;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.Engine.MachineLearning
+{
+    public static partial class Compute
+    {
+        /*************************************/
+        /**** Public Methods              ****/
+        /*************************************/
+
+        [Description("Reads a csv file and loads it into a Tensor primitive.")]
+        [Input("path", "The path to the csv file. If using a relative path, the root will be C:\\ProgramData\\BHoM")]
```

Comment:
Do we need the functionality of a relative path?

Reply:
To load datasets shipped with the experiments